src/net/peacefulcraft/sco/swordskills/utilities/DirectionalUtil.java

```diff
@@ -0,0 +1,112 @@
+package net.peacefulcraft.sco.swordskills.utilities;
+
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerMoveEvent;
+import org.bukkit.util.Vector;
+
+import net.peacefulcraft.sco.gamehandle.GameManager;
+import net.peacefulcraft.sco.gamehandle.player.SCOPlayer;
+
+public class DirectionalUtil implements Listener{
+    /**Player Directions */
+    public enum Movement {
+        BACKWARD, FORWARD_SPRINT, FORWARD, LEFT, RIGHT, STANDING, ROTATION, SIDE;
+    }
+
+    /**Cardinal Directions */
+    public enum CardinalDirection {
+        NORTH, NORTH_EAST, EAST, SOUTH_EAST, SOUTH, SOUTH_WEST, WEST, NORTH_WEST;
+    }
+
+    @EventHandler
```

Comment:
Where all are we going to need to use these values? If we're going to use them in SwordSkill checks / execution calls we'd probably want to add
```suggestion
    @EventHandler(priority = EventPriority.HIGHEST)
```
so that this gets processed before any SwordSkill code happens.